src/wazuh_db/helpers/wdb_agents_helpers.c

```diff
@@ -0,0 +1,115 @@
+/*
+ * Wazuh SQLite integration
+ * Copyright (C) 2015-2021, Wazuh Inc.
+ * February 17, 2021.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation.
+ */
+
+#include "wdb_agents_helpers.h"
+//#include "defs.h"
```

Comment:
If this define is no longer necessary, could be removed.

Reply:
Fixed